dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -883,6 +887,38 @@ public void download(final List<String> args) {
         }
     }
 
+    /**
+     * Validates that any JSON and/or YAML files are syntactically valid.
+     *
+     * @param args
+     */
+    protected void preValidateLaunchArguments(List<String> args) {
+        // Create a copy of args for prevalidation since optVal removes args from list
```

Comment:
1. It's optVals that removes the args from the list -- it is called from optVal.
2. You might want to give more context, either here, or in the JavaDoc about this behavior and in general, i.e., it may not be immediately apparent why all the the other calls optVal in this file don't need to do this.

Maybe something like this, which needs to be tweaked:
```
Validates that any JSON and/or YAML files being passed in are syntactically valid. If there is any error other than invalid syntax, the error is ignored and expected to be handled later. Because of this the args need to be preserved for the later handling
```